examples/workdir/inventory/hosts.yml

```diff
@@ -18,3 +18,23 @@ osh-deployer:
 ses_nodes:
   vars:
     ansible_user: root
+
+# added for airship
+airship-openstack-compute-workers:
+
+airship-openstack-control-workers:
+
+airship-ucp-workers:
+
+all:
```

Comment:
all is a meta group and should not be redefined.

Reply:
Thx! Will remove